docs/self-hosted/install/install-on-kubernetes.md

```diff
@@ -4,45 +4,46 @@ url: /docs/self-hosted/latest/install/install-on-kubernetes/
 # Install Gitpod Self-Hosted on Kubernetes
 
-This section describes how to install Gitpod on any Kubernetes cluster.
-Gitpod also provides more optimized installations offering better performance for particular cloud providers:
-* *Google Cloud Platform*: Install Gitpod in a blank GCP project [using a script that automates the procedure](../install-on-gcp-script/).
-* *AWS*: Install Gitpod on AWS [using a script that automates the procedure](../install-on-aws-script/).
+This section describes how to install Gitpod on any Kubernetes cluster using [Helm](https://helm.sh). This ist the most flexible and generic way of installing Gitpod.  
 
-Gitpod is installed using [Helm](https://helm.sh).
-The source of the chart can be found [here](https://github.com/gitpod-io/gitpod/blob/master/chart/).
-Releases are published at https://charts.gitpod.io.
+For some cloud providers we offer [Terraform](https://www.terraform.io/) scripts packaged into an installer that set up the infrastructure around Gitpod following performance-relevant best practices and using the cloud provider's managed services: Kubernetes cluster, database, storage, image registry. We recommend using the installer if available for your cloud provider. Once the installer has created the cluster, it will pre-configure and install this Gitpod helm chart into it.
+* [Installer for Google Cloud Platform (GCP)](../install-on-gcp-script/).
+* [Installer for Amazon Web Services (AWS)](../install-on-aws-script/).
 
 ## Prerequisites
- 1. Kubernetes version 1.15 <= x <= 1.17 is supported
+ 
+ 1. A Kubernetes cluster in version 1.15 <= x <= 1.17. 
+ 
+ 2. lcoal `kubectl` with connection to your cluster
+ 
+ 3. local `helm` in version >= 3.
 
- > Note: Gitpod should work on small nodes out of the box (2vCPUs, 8GB RAM). For a solid experience we recommend at least 4vCPUs and 16GB RAM for workspaces nodes.
+ Gitpod should work on small Kubernetes nodes out of the box (2vCPUs, 8GB RAM). For a better experience we recommend at least 4vCPUs and 16GB RAM for workspaces nodes. For cost efficienty, we recommend to enable cluster-autoscaling.
 
 ## Installation
 
-To initiate the deployment run the following commands:
+To install Gitpod in your Kubernetes cluster follow these steps:
+
+Run the following commands in your local terminal:
 ```console
 helm repo add gitpod.io https://charts.gitpod.io
 
 helm install gitpod gitpod.io/gitpod
 ```
 
- > Review the deployment worked properly by running `kubectl get pods`. Eventually all pods should be up-and-running. In case they are not have a look the the [Troubleshooting Guide](../troubleshooting/)
- 
- 1. Configure [ingress into the cluster](../configure-ingress/).
-    It makes sense to create a new folder for all Gitpod specific custom configuration and files:
-    ```
-    mkdir gitpod && cd gitpod
-    ```
+Now run `kubectl get pods` and verify that all pods are in state `RUNNING`. If some are not, please see the [Troubleshooting Guide](../troubleshooting/).
```

Comment:
I _really_ like the numbering (ease parsing and guides the eye when moving back-and-forth between terminal and doc like here).

Reply:
re-added. But it may not be rendered properly by Gatsby.